invenio_users_resources/resources/users/resource.py

```diff
@@ -44,6 +46,7 @@ def create_url_rules(self):
             route("POST", routes["deactivate"], self.deactivate),
             route("POST", routes["activate"], self.activate),
             route("POST", routes["impersonate"], self.impersonate),
+            route("POST", routes["create-via-admin"], self.create_via_admin),
```

Comment:
I think using the list endpoint here would be better to make it similar to other endpoints, like records or communities.